fallout/items/clothing/suits/armor.dm

```diff
@@ -207,47 +207,59 @@
 	inhand_icon_state = "ncr_reinforced_mantle_w"
 	armor = list("melee" = 50, "bullet" = 45, "laser" = 25, "energy" = 25, "bomb" = 30, "bio" = 25, "rad" = 25, "fire" = 30, "acid" = 25)
 
-/obj/item/clothing/suit/armor/fallout/legion
```

Comment:
You should define the base /legion type as something, even if it's just placeholder.